app/views/chatrooms/show.html.erb

```diff
@@ -1,5 +1,4 @@
-<div class="row col-md-8 col-md-offset-2">
-  <h1><%= @chatroom.topic %></h1>
+<h1><%= @chatroom.topic %></h1>
```

Comment:
same as with Chatrooms index. 

Not a blocker, just worth noting. It'll be fixed when we dive in on styling.